app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,211 @@
+package agency.five.codebase.android.movieapp.ui.moviedetails
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.ui.component.ActorCard
+import agency.five.codebase.android.movieapp.ui.component.CircularProgressBar
+import agency.five.codebase.android.movieapp.ui.component.CrewItem
+import agency.five.codebase.android.movieapp.ui.component.FavoriteButton
+import agency.five.codebase.android.movieapp.ui.moviedetails.mapper.MovieDetailsMapper
+import agency.five.codebase.android.movieapp.ui.moviedetails.mapper.MovieDetailsMapperImpl
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+
+private val MovieDetailsMapper: MovieDetailsMapper = MovieDetailsMapperImpl()
+
+val movieDetailsViewState = MovieDetailsMapper.toMovieDetailsViewState(MoviesMock.getMovieDetails())
+
+@Composable
+fun MovieDetailsRoute(
+    //actions
+) {
+    val detailsViewState by remember { mutableStateOf(movieDetailsViewState) }
+
+    MovieDetailsScreen(
+        detailsViewState,
+    )
+}
+
+@Composable
+fun MovieDetailsScreen(
+    movieDetailsViewState: MovieDetailsViewState,
+) {
+    LazyColumn(
+        modifier = Modifier
+            .background(color = MaterialTheme.colors.background)
+    ) {
+        item {
+            MovieImage(movieDetailsViewState)
+        }
+
+        item {
+            MovieOverview(movieDetailsViewState)
+        }
+
+        item {
+            MovieCrewman(movieDetailsViewState)
+        }
+
+        item {
+            MovieCast(movieDetailsViewState)
+        }
+    }
```

Comment:
This is not use case for `LazyColumn`, use `Column` with scrolling enabled instead.